smart_tagger/__init__.py

```diff
@@ -0,0 +1,10 @@
+##############################################################################
```

Comment:
See short copyright format in https://github.com/OCA/maintainer-tools/tree/master/template/module